package.json

```diff
@@ -75,7 +75,6 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-test-loader": "3.0.0",
     "ember-cli-uglify": "3.0.0",
-    "ember-cli-yuidoc": "0.9.1",
```

Comment:
Shouldn't the "yuidoc-ember-cli-theme" dependency be removed from the project as well?

Reply:
Done.